notebooks/book1/02/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    ax[0].set_title("PDF of Uniform Distribution")
</pre><p>Try to avoid long title, it will make figure crowdy .just write "Uniform distribution" (d is not capital)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>